src/prometheus.rs

```diff
@@ -161,6 +161,13 @@ mod hidden {
 		register_gauge!(opts!("staking_miner_solution_weight", "Weight of the solution submitted"))
 			.unwrap()
 	});
+	static ERA: Lazy<Gauge> = Lazy::new(|| {
+		register_gauge!(opts!(
+			"staking_miner_era_hours",
```

Comment:
@BulatSaif any preferred naming of this?

You wrote in the issue session length but I guess we want the actual "Era period/interval" here